test/test-network/systemd-networkd-tests.py

```diff
@@ -838,6 +839,7 @@ class NetworkdNetDevTests(unittest.TestCase, Utilities):
         '21-vlan.network',
         '25-6rd-tunnel.netdev',
         '25-bareudp.netdev',
+        '25-batadv.netdev',
```

Comment:
Oh, I noticed test config is added but no test for it is added. Could you add a brief test?

Reply:
I am not sure if this would work as the test machines might not have the batadv module in the correct version.

Reply:
I guess you can use `expectedFailureIfModuleIsNotAvailable()`, maybe e.g., test_wireguard() can help you.

Reply:
How can I trigger the test on my local machine without running all other network tests?

Reply:
`sudo ./systemd-networkd-tests.py --build-dir=BUILD_DIR_YOU_USE NetworkdNetDevTests.test_batadv`